Hush/Assets/Prefabs/Environment/Domes/Relic Dome.prefab

```diff
@@ -0,0 +1,142 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &4256472995991901031
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 3484052249403165149}
+  - component: {fileID: 573659860359218808}
+  - component: {fileID: 2311186444272300722}
+  - component: {fileID: 7786904841737782451}
+  - component: {fileID: 6917358588575371276}
+  - component: {fileID: 4105969258735876799}
+  - component: {fileID: 4816975685772881064}
+  m_Layer: 9
+  m_Name: Relic Dome
+  m_TagString: Relic Dome
+  m_Icon: {fileID: 0}
+  m_NavMeshLayer: 0
+  m_StaticEditorFlags: 0
+  m_IsActive: 1
+--- !u!4 &3484052249403165149
+Transform:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 4256472995991901031}
+  m_LocalRotation: {x: -0.7071068, y: 0, z: -0, w: 0.7071067}
```

Comment:
### Minor
Can you reset the Transform component in the prefab, The prefab isn't centered on itself. Set the scale to a round number (like 525) and set the X rotation to -90 exactly.

Also possibly add an empty parent GO for the dome. That way when you place it into a scene the prefab appears to have no rotation in the inspector